File: fe-destiny-community/src/app/user/service/UserModel.ts
export interface UserModel {
    type: MessageType;
    user_id: number;
    username: string;
    fullname: string;
    email: string;
    avatar: string;
    messageUnRead: number;
    lastMessage: string;
    online: Date |null;
    isFriend:boolean;
    status: boolean;
  }
  
  export enum MessageType {
    JOIN = 'JOIN',
    LEAVE = 'LEAVE'
  }
  

File: be-destiny-community/BE_Destiny/src/main/java/com/davisy/model/chat/UserModel.java
package com.davisy.model.chat;

import java.awt.TrayIcon.MessageType;
import java.util.Calendar;
import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserModel {
	private MessageType type;
	private int user_id;
	private String username;
	private String fullname;
	private String email;
	private String avatar;
	private int messageUnRead;
	private String lastMessage;
	private Calendar online;
	private boolean isFriend;
	private boolean hide ;
	private boolean status;
	

	public enum MessageType {
		JOIN, LEAVE
	}
}
